src/comments.js

```diff
@@ -1,74 +1,157 @@
 const core = require('@actions/core');
 const github = require('@actions/github');
 
-const owlberts = {
-  create: 'https://user-images.githubusercontent.com/313895/167224035-b34efcd6-854e-4cb4-92bd-10d717d2a6b1.png',
-  update: 'https://user-images.githubusercontent.com/313895/167228091-63ba7dff-c41f-4359-bd59-10e7ed3972aa.png',
-  delete: 'https://user-images.githubusercontent.com/313895/167224468-051b838c-19fd-4133-97a5-7e73ce9dc366.png',
-};
+const owlbert = 'https://user-images.githubusercontent.com/313895/167224035-b34efcd6-854e-4cb4-92bd-10d717d2a6b1.png';
 
-async function postComment(body) {
-  const token = core.getInput('github_token', { required: false });
-  if (!token) {
-    core.info('GitHub API token not present, not commenting on this pull request');
-    return undefined;
+function getFormattedDate() {
+  const options = { weekday: 'short', year: undefined, month: 'short', day: 'numeric', timeZone: 'America/New_York' };
+  const dateString = new Date().toLocaleString('en-US', options);
+  // toLocaleString() with those options returns "Wed, May 25" but I'm pedantic
+  return dateString.replace(',', '');
+}
+
+/*
+ * Generates a stub with empty implementations of the Octokit functions we use
+ * here. This is helpful so that we don't have to have a lot of if-else logic
+ * throughout this module.
+ */
+function getFakeOctokit() {
+  return {
+    rest: {
+      issues: {
+        createComment: () => {},
+        updateComment: () => {},
+        listComments: () => {},
+      },
+    },
+  };
+}
+
+let octokit;
```

Comment:
I'm surprised that this being here isn't throwing any parse or ESLint errors on line 55 where you do `const octokit`.

Reply:
So am I 🤷🏼  I'll rename this to `memoizedOctokit` (unless you've got a better idea?)

Reply:
renaming it should be good enough